### ISSUE TITLE
1.1 preview 1 breaks NET5.0 and 6.0 projects.

### ISSUE THREAD

**Issue by dhoehna** [microsoft/WindowsAppSDK#2338]
### Describe the bug

Adding Windows App SDK 1.1-Preview 1 to a .NET projects causes the project build to fail, saying that "Target Framework is empty".  Specifically: A numeric comparison was attempted on "$(TargetPlatformVersion)" that evaluates to "" instead of a number, in condition "'$(TargetPlatformVersion)' < '10.0.18362.0'".  C:\Users\dahoehna\.nuget\packages\microsoft.windowsappsdk\1.1.0-preview1\buildTransitive\Microsoft.InteractiveExperiences.Common.targets

### Steps to reproduce the bug

1. Make a new C# project targeting either 5.0 or 6.0.
2. Install the nuget package for Windows App SDK 1.1-Preview 1.

### Expected behavior

A message in the build output asking to add the windows version to the "TargetFramework" attribute in the csproj.

### Packaging type

Unpackaged

### IDE

Visual Studio 2022

### Additional context

There is currently a workaround by adding the version manually.

Change <TargetFramework>net6.0</TargetFramework> to <TargetFramework>net6.0-windows10.0.19041.0</TargetFramework>

**Comment by riverar** [comment]
Yup, I run into this brick wall a lot, to the point where I learned to check TFM when _strange things_ start happening. When this occurs you typically lose access to the project file editing experience too, making it very difficult to fix. Generally Visual Studio requires a restart once in this state.

**Comment by jeromelaban** [comment]
This issue is still very much present, but can be aggravated in cross-targeted projects by the NuGet package manager not supporting conditional ItemGroups properly: https://developercommunity.visualstudio.com/t/NuGet-Package-manager-does-not-honor-con/10269291

**Comment by MSLukeWest** [comment]
@manodasanW There's been another instance of this, see https://developercommunity.visualstudio.com/t/installing-WindowsAppSDK-can-lead-to-pro/10429073

Can the targets be updated to avoid this comparison when TargetPlatformVersion isn't set?

**Comment by axtens** [comment]
This is still an issue when converting from net7.0 to net8.0 in Uno projects, viz
```
A numeric comparison was attempted on "$(TargetPlatformVersion)" that evaluates to "" instead of a number, in condition "'$(TargetPlatformIdentifier)' == 'Windows' and
                                              '$(TargetPlatformVersion)' >= '8.0'".  C:\Users\Daisy\.nuget\packages\microsoft.windowsappsdk\1.5.240607001\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets
```

**Comment by ramiz272** [comment]
replace this line in file : C:\Users\dahoehna.nuget\packages\microsoft.windowsappsdk\1.1.0-preview1\buildTransitive\Microsoft.InteractiveExperiences.Common.targets

**Line to replace:**
`<Ixp-UAPTargetVersion Condition="'$(TargetPlatformVersion)' &lt; '10.0.18362.0'">uap10.0.17763</Ixp-UAPTargetVersion>`
**TO:**
`<Ixp-UAPTargetVersion Condition=" '$(TargetPlatformVersion)' != '' And '$(TargetPlatformVersion)' &lt; '10.0.18362.0'">uap10.0.17763</Ixp-UAPTargetVersion>`

**WILL RESOLVE THIS ISSUE :)**

**Comment by ramiz272** [comment]
> This is still an issue when converting from net7.0 to net8.0 in Uno projects, viz
> 
> ```
> A numeric comparison was attempted on "$(TargetPlatformVersion)" that evaluates to "" instead of a number, in condition "'$(TargetPlatformIdentifier)' == 'Windows' and
>  '$(TargetPlatformVersion)' >= '8.0'".  C:\Users\Daisy\.nuget\packages\microsoft.windowsappsdk\1.5.240607001\buildTransitive\Microsoft.UI.Xaml.Markup.Compiler.interop.targets
> ```

replace this line in file : C:\Users\dahoehna.nuget\packages\microsoft.windowsappsdk\1.1.0-preview1\buildTransitive\Microsoft.InteractiveExperiences.Common.targets

**Line to replace:**
`<Ixp-UAPTargetVersion Condition="'$(TargetPlatformVersion)' &lt; '10.0.18362.0'">uap10.0.17763</Ixp-UAPTargetVersion>`
**TO:**
`<Ixp-UAPTargetVersion Condition=" '$(TargetPlatformVersion)' != '' And '$(TargetPlatformVersion)' &lt; '10.0.18362.0'">uap10.0.17763</Ixp-UAPTargetVersion>`

**WILL RESOLVE THIS ISSUE :)**

**Comment by LegendaryBlair** [comment]
The issue was fixed in WinAppSDK 1.5 and above, hence closing the issue.
